## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 434265`
Commit: `2e01bd903934d92260c3a4bd5e9d9b49d189a0cc`

---
### `aarch64-darwin`
<details>
  <summary>:x: 22 packages failed to build:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python312Packages.k-diffusion</li>
    <li>python312Packages.k-diffusion.dist</li>
    <li>python312Packages.lifelines</li>
    <li>python312Packages.lifelines.dist</li>
    <li>python312Packages.llama-index-embeddings-huggingface</li>
    <li>python312Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python312Packages.smolagents</li>
    <li>python312Packages.smolagents.dist</li>
    <li>python313Packages.k-diffusion</li>
    <li>python313Packages.k-diffusion.dist</li>
    <li>python313Packages.lifelines</li>
    <li>python313Packages.lifelines.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.smolagents</li>
    <li>python313Packages.smolagents.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 699 packages built:</summary>
  <ul>
    <li>acquire (python313Packages.acquire)</li>
    <li>acquire.dist (python313Packages.acquire.dist)</li>
    <li>aider-chat-with-browser</li>
    <li>aider-chat-with-browser.dist</li>
    <li>apkleaks</li>
    <li>apkleaks.dist</li>
    <li>azure-cli-extensions.rdbms-connect</li>
    <li>azure-cli-extensions.rdbms-connect.dist</li>
    <li>barman</li>
    <li>barman.dist</li>
    <li>beets (beetsPackages.beets, beetsPackages.beets-stable)</li>
    <li>beets-unstable (beetsPackages.beets-unstable)</li>
    <li>beets-unstable.dist (beetsPackages.beets-unstable.dist)</li>
    <li>beets-unstable.doc (beetsPackages.beets-unstable.doc)</li>
    <li>beets-unstable.man (beetsPackages.beets-unstable.man)</li>
    <li>beets.dist (beetsPackages.beets-stable.dist, beetsPackages.beets.dist)</li>
    <li>beets.doc (beetsPackages.beets-stable.doc, beetsPackages.beets.doc)</li>
    <li>beets.man (beetsPackages.beets-stable.man, beetsPackages.beets.man)</li>
    <li>browsr</li>
    <li>browsr.dist</li>
    <li>checkov</li>
    <li>checkov.dist</li>
    <li>cve-bin-tool</li>
    <li>cve-bin-tool.dist</li>
    <li>das</li>
    <li>das.dist</li>
    <li>deeptools</li>
    <li>deeptools.dist</li>
    <li>dvc (python313Packages.dvc)</li>
    <li>dvc-with-remotes</li>
    <li>dvc-with-remotes.dist</li>
    <li>dvc.dist (python313Packages.dvc.dist)</li>
    <li>flattenReferencesGraph</li>
    <li>flattenReferencesGraph.dist</li>
    <li>froide-govplan</li>
    <li>froide-govplan.dist</li>
    <li>harlequin</li>
    <li>harlequin.dist</li>
    <li>jadx</li>
    <li>marimo (python313Packages.marimo)</li>
    <li>marimo.dist (python313Packages.marimo.dist)</li>
    <li>minari (python313Packages.minari)</li>
    <li>minari.dist (python313Packages.minari.dist)</li>
    <li>multiqc</li>
    <li>multiqc.dist</li>
    <li>mycli</li>
    <li>mycli.dist</li>
    <li>netbox (netbox_4_3)</li>
    <li>pianotrans</li>
    <li>pianotrans.dist</li>
    <li>prowler</li>
    <li>prowler.dist</li>
    <li>py-wacz (python313Packages.wacz)</li>
    <li>py-wacz.dist (python313Packages.wacz.dist)</li>
    <li>python312Packages.altair</li>
    <li>python312Packages.altair.dist</li>
    <li>python312Packages.ax-platform</li>
    <li>python312Packages.ax-platform.dist</li>
    <li>python312Packages.backtesting</li>
    <li>python312Packages.backtesting.dist</li>
    <li>python312Packages.bash-kernel</li>
    <li>python312Packages.bash-kernel.dist</li>
    <li>python312Packages.bokeh</li>
    <li>python312Packages.bokeh.dist</li>
    <li>python312Packages.celery</li>
    <li>python312Packages.celery-batches</li>
    <li>python312Packages.celery-batches.dist</li>
    <li>python312Packages.celery-redbeat</li>
    <li>python312Packages.celery-redbeat.dist</li>
    <li>python312Packages.celery-singleton</li>
    <li>python312Packages.celery-singleton.dist</li>
    <li>python312Packages.celery.dist</li>
    <li>python312Packages.chart-studio</li>
    <li>python312Packages.chart-studio.dist</li>
    <li>python312Packages.cloudpathlib</li>
    <li>python312Packages.cloudpathlib.dist</li>
    <li>python312Packages.coffea</li>
    <li>python312Packages.coffea.dist</li>
    <li>python312Packages.coiled</li>
    <li>python312Packages.coiled.dist</li>
    <li>python312Packages.compressai</li>
    <li>python312Packages.compressai.dist</li>
    <li>python312Packages.dalle-mini</li>
    <li>python312Packages.dalle-mini.dist</li>
    <li>python312Packages.dash</li>
    <li>python312Packages.dash-bootstrap-components</li>
    <li>python312Packages.dash-bootstrap-components.dist</li>
    <li>python312Packages.dash-bootstrap-templates</li>
    <li>python312Packages.dash-bootstrap-templates.dist</li>
    <li>python312Packages.dash.dist</li>
    <li>python312Packages.dbt-bigquery</li>
    <li>python312Packages.dbt-bigquery.dist</li>
    <li>python312Packages.dissect-cobaltstrike</li>
    <li>python312Packages.dissect-cobaltstrike.dist</li>
    <li>python312Packages.django-celery-beat</li>
    <li>python312Packages.django-celery-beat.dist</li>
    <li>python312Packages.django-celery-email</li>
    <li>python312Packages.django-celery-email.dist</li>
    <li>python312Packages.django-celery-results</li>
    <li>python312Packages.django-celery-results.dist</li>
    <li>python312Packages.django-filingcabinet</li>
    <li>python312Packages.django-filingcabinet.dist</li>
    <li>python312Packages.django-google-analytics-app</li>
    <li>python312Packages.django-google-analytics-app.dist</li>
    <li>python312Packages.django-health-check</li>
    <li>python312Packages.django-health-check.dist</li>
    <li>python312Packages.django-raster</li>
    <li>python312Packages.django-raster.dist</li>
    <li>python312Packages.django-storages</li>
    <li>python312Packages.django-storages.dist</li>
    <li>python312Packages.djmail</li>
    <li>python312Packages.djmail.dist</li>
    <li>python312Packages.duckdb</li>
    <li>python312Packages.duckdb-engine</li>
    <li>python312Packages.duckdb-engine.dist</li>
    <li>python312Packages.duckdb.dist</li>
    <li>python312Packages.dvc</li>
    <li>python312Packages.dvc-gs</li>
    <li>python312Packages.dvc-gs.dist</li>
    <li>python312Packages.dvc-hdfs</li>
    <li>python312Packages.dvc-hdfs.dist</li>
    <li>python312Packages.dvc-task</li>
    <li>python312Packages.dvc-task.dist</li>
    <li>python312Packages.dvc.dist</li>
    <li>python312Packages.dvclive</li>
    <li>python312Packages.dvclive.dist</li>
    <li>python312Packages.energyflow</li>
    <li>python312Packages.energyflow.dist</li>
    <li>python312Packages.experiment-utilities</li>
    <li>python312Packages.experiment-utilities.dist</li>
    <li>python312Packages.explorerscript</li>
    <li>python312Packages.explorerscript.dist</li>
    <li>python312Packages.fastai</li>
    <li>python312Packages.fastai.dist</li>
    <li>python312Packages.fastjet</li>
    <li>python312Packages.fastjet.dist</li>
    <li>python312Packages.filedepot</li>
    <li>python312Packages.filedepot.dist</li>
    <li>python312Packages.firebase-admin</li>
    <li>python312Packages.firebase-admin.dist</li>
    <li>python312Packages.flow-record</li>
    <li>python312Packages.flow-record.dist</li>
    <li>python312Packages.flower</li>
    <li>python312Packages.flower.dist</li>
    <li>python312Packages.formulaic</li>
    <li>python312Packages.formulaic.dist</li>
    <li>python312Packages.frictionless</li>
    <li>python312Packages.frictionless.dist</li>
    <li>python312Packages.gcp-storage-emulator</li>
    <li>python312Packages.gcp-storage-emulator.dist</li>
    <li>python312Packages.gcsfs</li>
    <li>python312Packages.gcsfs.dist</li>
    <li>python312Packages.google-cloud-automl</li>
    <li>python312Packages.google-cloud-automl.dist</li>
    <li>python312Packages.google-cloud-bigquery</li>
    <li>python312Packages.google-cloud-bigquery.dist</li>
    <li>python312Packages.google-cloud-storage</li>
    <li>python312Packages.google-cloud-storage.dist</li>
    <li>python312Packages.gradio</li>
    <li>python312Packages.gradio.dist</li>
    <li>python312Packages.great-tables</li>
    <li>python312Packages.great-tables.dist</li>
    <li>python312Packages.harlequin-bigquery</li>
    <li>python312Packages.harlequin-bigquery.dist</li>
    <li>python312Packages.holoviews</li>
    <li>python312Packages.holoviews.dist</li>
    <li>python312Packages.hvplot</li>
    <li>python312Packages.hvplot.dist</li>
    <li>python312Packages.ibis-framework</li>
    <li>python312Packages.ibis-framework.dist</li>
    <li>python312Packages.igraph</li>
    <li>python312Packages.igraph.dist</li>
    <li>python312Packages.intake</li>
    <li>python312Packages.intake.dist</li>
    <li>python312Packages.iplotx</li>
    <li>python312Packages.iplotx.dist</li>
    <li>python312Packages.itables</li>
    <li>python312Packages.itables.dist</li>
    <li>python312Packages.jupysql</li>
    <li>python312Packages.jupysql.dist</li>
    <li>python312Packages.kbcstorage</li>
    <li>python312Packages.kbcstorage.dist</li>
    <li>python312Packages.kmapper</li>
    <li>python312Packages.kmapper.dist</li>
    <li>python312Packages.lancedb</li>
    <li>python312Packages.lancedb.dist</li>
    <li>python312Packages.langchain-community</li>
    <li>python312Packages.langchain-community.dist</li>
    <li>python312Packages.langchain-experimental</li>
    <li>python312Packages.langchain-experimental.dist</li>
    <li>python312Packages.leidenalg</li>
    <li>python312Packages.leidenalg.dist</li>
    <li>python312Packages.librosa</li>
    <li>python312Packages.librosa.dist</li>
    <li>python312Packages.livelossplot</li>
    <li>python312Packages.livelossplot.dist</li>
    <li>python312Packages.llama-cloud-services</li>
    <li>python312Packages.llama-cloud-services.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-agent-openai</li>
    <li>python312Packages.llama-index-agent-openai.dist</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-core</li>
    <li>python312Packages.llama-index-core.dist</li>
    <li>python312Packages.llama-index-embeddings-gemini</li>
    <li>python312Packages.llama-index-embeddings-gemini.dist</li>
    <li>python312Packages.llama-index-embeddings-google</li>
    <li>python312Packages.llama-index-embeddings-google.dist</li>
    <li>python312Packages.llama-index-embeddings-ollama</li>
    <li>python312Packages.llama-index-embeddings-ollama.dist</li>
    <li>python312Packages.llama-index-embeddings-openai</li>
    <li>python312Packages.llama-index-embeddings-openai.dist</li>
    <li>python312Packages.llama-index-graph-stores-nebula</li>
    <li>python312Packages.llama-index-graph-stores-nebula.dist</li>
    <li>python312Packages.llama-index-graph-stores-neo4j</li>
    <li>python312Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python312Packages.llama-index-graph-stores-neptune</li>
    <li>python312Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python312Packages.llama-index-legacy</li>
    <li>python312Packages.llama-index-legacy.dist</li>
    <li>python312Packages.llama-index-llms-ollama</li>
    <li>python312Packages.llama-index-llms-ollama.dist</li>
    <li>python312Packages.llama-index-llms-openai</li>
    <li>python312Packages.llama-index-llms-openai-like</li>
    <li>python312Packages.llama-index-llms-openai-like.dist</li>
    <li>python312Packages.llama-index-llms-openai.dist</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python312Packages.llama-index-program-openai</li>
    <li>python312Packages.llama-index-program-openai.dist</li>
    <li>python312Packages.llama-index-question-gen-openai</li>
    <li>python312Packages.llama-index-question-gen-openai.dist</li>
    <li>python312Packages.llama-index-readers-database</li>
    <li>python312Packages.llama-index-readers-database.dist</li>
    <li>python312Packages.llama-index-readers-file</li>
    <li>python312Packages.llama-index-readers-file.dist</li>
    <li>python312Packages.llama-index-readers-json</li>
    <li>python312Packages.llama-index-readers-json.dist</li>
    <li>python312Packages.llama-index-readers-llama-parse</li>
    <li>python312Packages.llama-index-readers-llama-parse.dist</li>
    <li>python312Packages.llama-index-readers-s3</li>
    <li>python312Packages.llama-index-readers-s3.dist</li>
    <li>python312Packages.llama-index-readers-twitter</li>
    <li>python312Packages.llama-index-readers-twitter.dist</li>
    <li>python312Packages.llama-index-readers-txtai</li>
    <li>python312Packages.llama-index-readers-txtai.dist</li>
    <li>python312Packages.llama-index-readers-weather</li>
    <li>python312Packages.llama-index-readers-weather.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index-vector-stores-google</li>
    <li>python312Packages.llama-index-vector-stores-google.dist</li>
    <li>python312Packages.llama-index-vector-stores-postgres</li>
    <li>python312Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python312Packages.llama-index-vector-stores-qdrant</li>
    <li>python312Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llama-parse</li>
    <li>python312Packages.llama-parse.dist</li>
    <li>python312Packages.marimo</li>
    <li>python312Packages.marimo.dist</li>
    <li>python312Packages.minari</li>
    <li>python312Packages.minari.dist</li>
    <li>python312Packages.msticpy</li>
    <li>python312Packages.msticpy.dist</li>
    <li>python312Packages.narwhals</li>
    <li>python312Packages.narwhals.dist</li>
    <li>python312Packages.nbsmoke</li>
    <li>python312Packages.nbsmoke.dist</li>
    <li>python312Packages.netbox-bgp</li>
    <li>python312Packages.netbox-bgp.dist</li>
    <li>python312Packages.netbox-documents</li>
    <li>python312Packages.netbox-documents.dist</li>
    <li>python312Packages.netbox-plugin-prometheus-sd</li>
    <li>python312Packages.netbox-plugin-prometheus-sd.dist</li>
    <li>python312Packages.netbox-reorder-rack</li>
    <li>python312Packages.netbox-reorder-rack.dist</li>
    <li>python312Packages.netbox-routing</li>
    <li>python312Packages.netbox-routing.dist</li>
    <li>python312Packages.neurokit2</li>
    <li>python312Packages.neurokit2.dist</li>
    <li>python312Packages.niaarm</li>
    <li>python312Packages.niaarm.dist</li>
    <li>python312Packages.opentelemetry-instrumentation-celery</li>
    <li>python312Packages.opentelemetry-instrumentation-celery.dist</li>
    <li>python312Packages.optuna</li>
    <li>python312Packages.optuna-dashboard</li>
    <li>python312Packages.optuna-dashboard.dist</li>
    <li>python312Packages.optuna.dist</li>
    <li>python312Packages.pandera</li>
    <li>python312Packages.pandera.dist</li>
    <li>python312Packages.panel</li>
    <li>python312Packages.panel.dist</li>
    <li>python312Packages.papermill</li>
    <li>python312Packages.papermill.dist</li>
    <li>python312Packages.piano-transcription-inference</li>
    <li>python312Packages.piano-transcription-inference.dist</li>
    <li>python312Packages.pins</li>
    <li>python312Packages.pins.dist</li>
    <li>python312Packages.plotly</li>
    <li>python312Packages.plotly.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pyannote-pipeline</li>
    <li>python312Packages.pyannote-pipeline.dist</li>
    <li>python312Packages.pylance</li>
    <li>python312Packages.pylance.dist</li>
    <li>python312Packages.pymatgen</li>
    <li>python312Packages.pymatgen.dist</li>
    <li>python312Packages.reflex</li>
    <li>python312Packages.reflex-chakra</li>
    <li>python312Packages.reflex-chakra.dist</li>
    <li>python312Packages.reflex.dist</li>
    <li>python312Packages.resampy</li>
    <li>python312Packages.resampy.dist</li>
    <li>python312Packages.scikit-tda</li>
    <li>python312Packages.scikit-tda.dist</li>
    <li>python312Packages.shiny</li>
    <li>python312Packages.shiny.dist</li>
    <li>python312Packages.skops</li>
    <li>python312Packages.skops.dist</li>
    <li>python312Packages.skrl</li>
    <li>python312Packages.skrl.dist</li>
    <li>python312Packages.smart-open</li>
    <li>python312Packages.smart-open.dist</li>
    <li>python312Packages.snakemake</li>
    <li>python312Packages.snakemake-storage-plugin-s3</li>
    <li>python312Packages.snakemake-storage-plugin-s3.dist</li>
    <li>python312Packages.snakemake-storage-plugin-xrootd</li>
    <li>python312Packages.snakemake-storage-plugin-xrootd.dist</li>
    <li>python312Packages.snakemake.dist</li>
    <li>python312Packages.spacy</li>
    <li>python312Packages.spacy-curated-transformers</li>
    <li>python312Packages.spacy-curated-transformers.dist</li>
    <li>python312Packages.spacy-loggers</li>
    <li>python312Packages.spacy-loggers.dist</li>
    <li>python312Packages.spacy-lookups-data</li>
    <li>python312Packages.spacy-lookups-data.dist</li>
    <li>python312Packages.spacy-transformers</li>
    <li>python312Packages.spacy-transformers.dist</li>
    <li>python312Packages.spacy.dist</li>
    <li>python312Packages.sqlframe</li>
    <li>python312Packages.sqlframe.dist</li>
    <li>python312Packages.sqlglot</li>
    <li>python312Packages.sqlglot.dist</li>
    <li>python312Packages.streamlit</li>
    <li>python312Packages.streamlit.dist</li>
    <li>python312Packages.sumo</li>
    <li>python312Packages.sumo.dist</li>
    <li>python312Packages.svgdigitizer</li>
    <li>python312Packages.svgdigitizer.dist</li>
    <li>python312Packages.synergy</li>
    <li>python312Packages.synergy.dist</li>
    <li>python312Packages.tenant-schemas-celery</li>
    <li>python312Packages.tenant-schemas-celery.dist</li>
    <li>python312Packages.textacy</li>
    <li>python312Packages.textacy.dist</li>
    <li>python312Packages.textnets</li>
    <li>python312Packages.textnets.dist</li>
    <li>python312Packages.torch-audiomentations</li>
    <li>python312Packages.torch-audiomentations.dist</li>
    <li>python312Packages.torchcrepe</li>
    <li>python312Packages.torchcrepe.dist</li>
    <li>python312Packages.torchlibrosa</li>
    <li>python312Packages.torchlibrosa.dist</li>
    <li>python312Packages.vector</li>
    <li>python312Packages.vector.dist</li>
    <li>python312Packages.vega</li>
    <li>python312Packages.vega.dist</li>
    <li>python312Packages.wacz</li>
    <li>python312Packages.wacz.dist</li>
    <li>python312Packages.wandb</li>
    <li>python312Packages.wandb.dist</li>
    <li>python312Packages.weasel</li>
    <li>python312Packages.weasel.dist</li>
    <li>python312Packages.whisperx</li>
    <li>python312Packages.whisperx.dist</li>
    <li>python313Packages.altair</li>
    <li>python313Packages.altair.dist</li>
    <li>python313Packages.ax-platform</li>
    <li>python313Packages.ax-platform.dist</li>
    <li>python313Packages.backtesting</li>
    <li>python313Packages.backtesting.dist</li>
    <li>python313Packages.bash-kernel</li>
    <li>python313Packages.bash-kernel.dist</li>
    <li>python313Packages.bokeh</li>
    <li>python313Packages.bokeh.dist</li>
    <li>python313Packages.celery</li>
    <li>python313Packages.celery-batches</li>
    <li>python313Packages.celery-batches.dist</li>
    <li>python313Packages.celery-redbeat</li>
    <li>python313Packages.celery-redbeat.dist</li>
    <li>python313Packages.celery-singleton</li>
    <li>python313Packages.celery-singleton.dist</li>
    <li>python313Packages.celery.dist</li>
    <li>python313Packages.chart-studio</li>
    <li>python313Packages.chart-studio.dist</li>
    <li>python313Packages.cloudpathlib</li>
    <li>python313Packages.cloudpathlib.dist</li>
    <li>python313Packages.coffea</li>
    <li>python313Packages.coffea.dist</li>
    <li>python313Packages.coiled</li>
    <li>python313Packages.coiled.dist</li>
    <li>python313Packages.compressai</li>
    <li>python313Packages.compressai.dist</li>
    <li>python313Packages.dash</li>
    <li>python313Packages.dash-bootstrap-components</li>
    <li>python313Packages.dash-bootstrap-components.dist</li>
    <li>python313Packages.dash-bootstrap-templates</li>
    <li>python313Packages.dash-bootstrap-templates.dist</li>
    <li>python313Packages.dash.dist</li>
    <li>python313Packages.dbt-bigquery</li>
    <li>python313Packages.dbt-bigquery.dist</li>
    <li>python313Packages.dissect</li>
    <li>python313Packages.dissect-cobaltstrike</li>
    <li>python313Packages.dissect-cobaltstrike.dist</li>
    <li>python313Packages.dissect-target</li>
    <li>python313Packages.dissect-target.dist</li>
    <li>python313Packages.dissect.dist</li>
    <li>python313Packages.django-celery-beat</li>
    <li>python313Packages.django-celery-beat.dist</li>
    <li>python313Packages.django-celery-email</li>
    <li>python313Packages.django-celery-email.dist</li>
    <li>python313Packages.django-celery-results</li>
    <li>python313Packages.django-celery-results.dist</li>
    <li>python313Packages.django-filingcabinet</li>
    <li>python313Packages.django-filingcabinet.dist</li>
    <li>python313Packages.django-google-analytics-app</li>
    <li>python313Packages.django-google-analytics-app.dist</li>
    <li>python313Packages.django-health-check</li>
    <li>python313Packages.django-health-check.dist</li>
    <li>python313Packages.django-raster</li>
    <li>python313Packages.django-raster.dist</li>
    <li>python313Packages.django-storages</li>
    <li>python313Packages.django-storages.dist</li>
    <li>python313Packages.djmail</li>
    <li>python313Packages.djmail.dist</li>
    <li>python313Packages.duckdb</li>
    <li>python313Packages.duckdb-engine</li>
    <li>python313Packages.duckdb-engine.dist</li>
    <li>python313Packages.duckdb.dist</li>
    <li>python313Packages.dvc-gs</li>
    <li>python313Packages.dvc-gs.dist</li>
    <li>python313Packages.dvc-hdfs</li>
    <li>python313Packages.dvc-hdfs.dist</li>
    <li>python313Packages.dvc-task</li>
    <li>python313Packages.dvc-task.dist</li>
    <li>python313Packages.dvclive</li>
    <li>python313Packages.dvclive.dist</li>
    <li>python313Packages.experiment-utilities</li>
    <li>python313Packages.experiment-utilities.dist</li>
    <li>python313Packages.explorerscript</li>
    <li>python313Packages.explorerscript.dist</li>
    <li>python313Packages.fastai</li>
    <li>python313Packages.fastai.dist</li>
    <li>python313Packages.fastjet</li>
    <li>python313Packages.fastjet.dist</li>
    <li>python313Packages.filedepot</li>
    <li>python313Packages.filedepot.dist</li>
    <li>python313Packages.firebase-admin</li>
    <li>python313Packages.firebase-admin.dist</li>
    <li>python313Packages.flow-record</li>
    <li>python313Packages.flow-record.dist</li>
    <li>python313Packages.flower</li>
    <li>python313Packages.flower.dist</li>
    <li>python313Packages.formulaic</li>
    <li>python313Packages.formulaic.dist</li>
    <li>python313Packages.frictionless</li>
    <li>python313Packages.frictionless.dist</li>
    <li>python313Packages.gcp-storage-emulator</li>
    <li>python313Packages.gcp-storage-emulator.dist</li>
    <li>python313Packages.gcsfs</li>
    <li>python313Packages.gcsfs.dist</li>
    <li>python313Packages.google-cloud-automl</li>
    <li>python313Packages.google-cloud-automl.dist</li>
    <li>python313Packages.google-cloud-bigquery</li>
    <li>python313Packages.google-cloud-bigquery.dist</li>
    <li>python313Packages.google-cloud-storage</li>
    <li>python313Packages.google-cloud-storage.dist</li>
    <li>python313Packages.gradio</li>
    <li>python313Packages.gradio.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.harlequin-bigquery</li>
    <li>python313Packages.harlequin-bigquery.dist</li>
    <li>python313Packages.holoviews</li>
    <li>python313Packages.holoviews.dist</li>
    <li>python313Packages.hvplot</li>
    <li>python313Packages.hvplot.dist</li>
    <li>python313Packages.ibis-framework</li>
    <li>python313Packages.ibis-framework.dist</li>
    <li>python313Packages.igraph</li>
    <li>python313Packages.igraph.dist</li>
    <li>python313Packages.intake</li>
    <li>python313Packages.intake.dist</li>
    <li>python313Packages.iplotx</li>
    <li>python313Packages.iplotx.dist</li>
    <li>python313Packages.itables</li>
    <li>python313Packages.itables.dist</li>
    <li>python313Packages.jupysql</li>
    <li>python313Packages.jupysql.dist</li>
    <li>python313Packages.kbcstorage</li>
    <li>python313Packages.kbcstorage.dist</li>
    <li>python313Packages.kmapper</li>
    <li>python313Packages.kmapper.dist</li>
    <li>python313Packages.lancedb</li>
    <li>python313Packages.lancedb.dist</li>
    <li>python313Packages.langchain-community</li>
    <li>python313Packages.langchain-community.dist</li>
    <li>python313Packages.langchain-experimental</li>
    <li>python313Packages.langchain-experimental.dist</li>
    <li>python313Packages.leidenalg</li>
    <li>python313Packages.leidenalg.dist</li>
    <li>python313Packages.librosa</li>
    <li>python313Packages.librosa.dist</li>
    <li>python313Packages.livelossplot</li>
    <li>python313Packages.livelossplot.dist</li>
    <li>python313Packages.llama-cloud-services</li>
    <li>python313Packages.llama-cloud-services.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-agent-openai</li>
    <li>python313Packages.llama-index-agent-openai.dist</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-gemini</li>
    <li>python313Packages.llama-index-embeddings-gemini.dist</li>
    <li>python313Packages.llama-index-embeddings-google</li>
    <li>python313Packages.llama-index-embeddings-google.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python313Packages.llama-index-program-openai</li>
    <li>python313Packages.llama-index-program-openai.dist</li>
    <li>python313Packages.llama-index-question-gen-openai</li>
    <li>python313Packages.llama-index-question-gen-openai.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-llama-parse</li>
    <li>python313Packages.llama-index-readers-llama-parse.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-postgres</li>
    <li>python313Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llama-parse</li>
    <li>python313Packages.llama-parse.dist</li>
    <li>python313Packages.msticpy</li>
    <li>python313Packages.msticpy.dist</li>
    <li>python313Packages.narwhals</li>
    <li>python313Packages.narwhals.dist</li>
    <li>python313Packages.nbsmoke</li>
    <li>python313Packages.nbsmoke.dist</li>
    <li>python313Packages.netbox-bgp</li>
    <li>python313Packages.netbox-bgp.dist</li>
    <li>python313Packages.netbox-documents</li>
    <li>python313Packages.netbox-documents.dist</li>
    <li>python313Packages.netbox-floorplan-plugin</li>
    <li>python313Packages.netbox-floorplan-plugin.dist</li>
    <li>python313Packages.netbox-plugin-prometheus-sd</li>
    <li>python313Packages.netbox-plugin-prometheus-sd.dist</li>
    <li>python313Packages.netbox-reorder-rack</li>
    <li>python313Packages.netbox-reorder-rack.dist</li>
    <li>python313Packages.netbox-routing</li>
    <li>python313Packages.netbox-routing.dist</li>
    <li>python313Packages.neurokit2</li>
    <li>python313Packages.neurokit2.dist</li>
    <li>python313Packages.niaarm</li>
    <li>python313Packages.niaarm.dist</li>
    <li>python313Packages.opentelemetry-instrumentation-celery</li>
    <li>python313Packages.opentelemetry-instrumentation-celery.dist</li>
    <li>python313Packages.optuna</li>
    <li>python313Packages.optuna-dashboard</li>
    <li>python313Packages.optuna-dashboard.dist</li>
    <li>python313Packages.optuna.dist</li>
    <li>python313Packages.pandera</li>
    <li>python313Packages.pandera.dist</li>
    <li>python313Packages.panel</li>
    <li>python313Packages.panel.dist</li>
    <li>python313Packages.papermill</li>
    <li>python313Packages.papermill.dist</li>
    <li>python313Packages.piano-transcription-inference</li>
    <li>python313Packages.piano-transcription-inference.dist</li>
    <li>python313Packages.pins</li>
    <li>python313Packages.pins.dist</li>
    <li>python313Packages.plotly</li>
    <li>python313Packages.plotly.dist</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pyannote-pipeline</li>
    <li>python313Packages.pyannote-pipeline.dist</li>
    <li>python313Packages.pylance</li>
    <li>python313Packages.pylance.dist</li>
    <li>python313Packages.reflex</li>
    <li>python313Packages.reflex-chakra</li>
    <li>python313Packages.reflex-chakra.dist</li>
    <li>python313Packages.reflex.dist</li>
    <li>python313Packages.resampy</li>
    <li>python313Packages.resampy.dist</li>
    <li>python313Packages.scikit-tda</li>
    <li>python313Packages.scikit-tda.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>python313Packages.skops</li>
    <li>python313Packages.skops.dist</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.skrl.dist</li>
    <li>python313Packages.smart-open</li>
    <li>python313Packages.smart-open.dist</li>
    <li>snakemake (python313Packages.snakemake)</li>
    <li>python313Packages.snakemake-storage-plugin-s3</li>
    <li>python313Packages.snakemake-storage-plugin-s3.dist</li>
    <li>python313Packages.snakemake-storage-plugin-xrootd</li>
    <li>python313Packages.snakemake-storage-plugin-xrootd.dist</li>
    <li>snakemake.dist (python313Packages.snakemake.dist)</li>
    <li>python313Packages.spacy</li>
    <li>python313Packages.spacy-curated-transformers</li>
    <li>python313Packages.spacy-curated-transformers.dist</li>
    <li>python313Packages.spacy-loggers</li>
    <li>python313Packages.spacy-loggers.dist</li>
    <li>python313Packages.spacy-lookups-data</li>
    <li>python313Packages.spacy-lookups-data.dist</li>
    <li>python313Packages.spacy-transformers</li>
    <li>python313Packages.spacy-transformers.dist</li>
    <li>python313Packages.spacy.dist</li>
    <li>python313Packages.sqlframe</li>
    <li>python313Packages.sqlframe.dist</li>
    <li>python313Packages.sqlglot</li>
    <li>python313Packages.sqlglot.dist</li>
    <li>streamlit (python313Packages.streamlit)</li>
    <li>streamlit.dist (python313Packages.streamlit.dist)</li>
    <li>python313Packages.svgdigitizer</li>
    <li>python313Packages.svgdigitizer.dist</li>
    <li>python313Packages.synergy</li>
    <li>python313Packages.synergy.dist</li>
    <li>python313Packages.tenant-schemas-celery</li>
    <li>python313Packages.tenant-schemas-celery.dist</li>
    <li>python313Packages.textacy</li>
    <li>python313Packages.textacy.dist</li>
    <li>python313Packages.torch-audiomentations</li>
    <li>python313Packages.torch-audiomentations.dist</li>
    <li>python313Packages.torchcrepe</li>
    <li>python313Packages.torchcrepe.dist</li>
    <li>python313Packages.torchlibrosa</li>
    <li>python313Packages.torchlibrosa.dist</li>
    <li>python313Packages.vector</li>
    <li>python313Packages.vector.dist</li>
    <li>python313Packages.vega</li>
    <li>python313Packages.vega.dist</li>
    <li>python313Packages.wandb</li>
    <li>python313Packages.wandb.dist</li>
    <li>python313Packages.weasel</li>
    <li>python313Packages.weasel.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>quark-engine</li>
    <li>quark-engine.dist</li>
    <li>tandoor-recipes</li>
    <li>theharvester</li>
    <li>theharvester.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python312Packages.lifelines</summary>
<pre>
lifelines/tests/utils/test_utils.py::test_qth_survival_time_accepts_a_model
  /private/tmp/nix-build-python3.12-lifelines-0.30.0.drv-0/source/lifelines/fitters/__init__.py:279: ApproximationWarning: Approximating using `survival_function_`. To increase accuracy, try using or increasing the resolution of the timeline kwarg in `.fit(..., timeline=timeline)`.
  
    warnings.warn(

lifelines/tests/utils/test_utils.py::test_rmst_works_with_return_variance
  /private/tmp/nix-build-python3.12-lifelines-0.30.0.drv-0/source/lifelines/utils/__init__.py:263: ApproximationWarning: Approximating RMST using the precomputed survival function. You likely will get a more accurate estimate if you provide the fitted Model instead of the survival function.
  
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
===Flaky Test Report===

test_rmst_approximate_solution passed 1 out of the required 1 times. Success!

===End Flaky Test Report===
=========================== short test summary info ============================
FAILED lifelines/tests/test_npmle.py::test_mice_scipy - AssertionError: 
==== 1 failed, 143 passed, 75 skipped, 1 deselected, 61 warnings in 31.06s =====</pre>
</details>
<details>
<summary>python313Packages.lifelines</summary>
<pre>
lifelines/tests/utils/test_utils.py::test_qth_survival_time_accepts_a_model
  /private/tmp/nix-build-python3.13-lifelines-0.30.0.drv-0/source/lifelines/fitters/__init__.py:279: ApproximationWarning: Approximating using `survival_function_`. To increase accuracy, try using or increasing the resolution of the timeline kwarg in `.fit(..., timeline=timeline)`.
  
    warnings.warn(

lifelines/tests/utils/test_utils.py::test_rmst_works_with_return_variance
  /private/tmp/nix-build-python3.13-lifelines-0.30.0.drv-0/source/lifelines/utils/__init__.py:263: ApproximationWarning: Approximating RMST using the precomputed survival function. You likely will get a more accurate estimate if you provide the fitted Model instead of the survival function.
  
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
===Flaky Test Report===

test_rmst_approximate_solution passed 1 out of the required 1 times. Success!

===End Flaky Test Report===
=========================== short test summary info ============================
FAILED lifelines/tests/test_npmle.py::test_mice_scipy - AssertionError: 
==== 1 failed, 143 passed, 75 skipped, 1 deselected, 61 warnings in 30.55s =====</pre>
</details>
</details>

--- a/pkgs/development/python-modules/google-cloud-storage/default.nix
+++ b/pkgs/development/python-modules/google-cloud-storage/default.nix
@@ -100,6 +100,6 @@ buildPythonPackage rec {
     homepage = "https://github.com/googleapis/python-storage";
     changelog = "https://github.com/googleapis/python-storage/blob/v${version}/CHANGELOG.md";
     license = lib.licenses.asl20;
-    maintainers = with lib.maintainers; [ ];
+    maintainers = with lib.maintainers; [ sarahec ];
   };
 }

--- a/pkgs/development/python-modules/google-cloud-storage/default.nix
+++ b/pkgs/development/python-modules/google-cloud-storage/default.nix
@@ -18,14 +18,14 @@
 
 buildPythonPackage rec {
   pname = "google-cloud-storage";
-  version = "3.2.0";
+  version = "3.3.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "googleapis";
     repo = "python-storage";
     tag = "v${version}";
-    hash = "sha256-05jPtpsEQYqReDaeLj/Vzkz761jZfLQXC20WjTIPVc8=";
+    hash = "sha256-I0wC/BV8fJr3JW1nyq2TPJZlZaT4+h2lJBdGTttSzRo=";
   };
 
   pythonRelaxDeps = [ "google-auth" ];
@@ -76,7 +76,6 @@ buildPythonPackage rec {
   ];
 
   disabledTestPaths = [
-    "samples/"
     "tests/unit/test_bucket.py"
     "tests/system/test_blob.py"
     "tests/system/test_bucket.py"

--- a/pkgs/development/python-modules/google-cloud-storage/default.nix
+++ b/pkgs/development/python-modules/google-cloud-storage/default.nix
@@ -1,7 +1,7 @@
 {
   lib,
   buildPythonPackage,
-  fetchPypi,
+  fetchFromGitHub,
   google-auth,
   google-cloud-core,
   google-cloud-iam,
@@ -21,12 +21,11 @@ buildPythonPackage rec {
   version = "3.2.0";
   pyproject = true;
 
-  disabled = pythonOlder "3.7";
-
-  src = fetchPypi {
-    pname = "google_cloud_storage";
-    inherit version;
-    hash = "sha256-3syoQwdgNvRWMxmMEl0YYf+/R+v1wOO5jcubLbFViWw=";
+  src = fetchFromGitHub {
+    owner = "googleapis";
+    repo = "python-storage";
+    tag = "v${version}";
+    hash = "sha256-05jPtpsEQYqReDaeLj/Vzkz761jZfLQXC20WjTIPVc8=";
   };
 
   pythonRelaxDeps = [ "google-auth" ];
@@ -52,6 +51,11 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  enabledTestPaths = [
+    "tests/unit/"
+    "tests/system/"
+  ];
+
   disabledTests = [
     # Disable tests which require credentials and network access
     "create"
@@ -72,6 +76,7 @@ buildPythonPackage rec {
   ];
 
   disabledTestPaths = [
+    "samples/"
     "tests/unit/test_bucket.py"
     "tests/system/test_blob.py"
     "tests/system/test_bucket.py"
@@ -90,11 +95,11 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "google.cloud.storage" ];
 
-  meta = with lib; {
+  meta = {
     description = "Google Cloud Storage API client library";
     homepage = "https://github.com/googleapis/python-storage";
     changelog = "https://github.com/googleapis/python-storage/blob/v${version}/CHANGELOG.md";
-    license = licenses.asl20;
-    maintainers = [ ];
+    license = lib.licenses.asl20;
+    maintainers = with lib.maintainers; [ ];
   };
 }

